### ISSUE TITLE
Fill alpha property is not available on export of animated vector drawable

### ISSUE THREAD

**Issue by SanthoshDhandapani** [alexjlockwood/ShapeShifter#215]
Fill alpha property is not available in the xml when the animation is exported as animated vector drawable file

**Comment by alexjlockwood** [comment]
Sorry, could you clarify the problem with an example?

It would be especially helpful if you could attach the `.shapeshifter` file that exhibits the problem (go to "File" --> "Save" and attach the `.shapeshifter` file to this issue).

When I go to "File" --> "Demo" and select "Heartbreak" (a demo which has a couple fillAlpha animation blocks), I am able to export the demo to AVD and I see the `fillAlpha` object animators in the exported result, so I'm not sure what the issue you are having could be.

**Comment by SanthoshDhandapani** [comment]
Hi alex thanks for the quick response.

This is the issue i am mentioning about
![ss_fill_alpha_property_miss](https://user-images.githubusercontent.com/10140204/32825172-e8cc2612-ca09-11e7-85b1-58a317bd1850.gif)

Attached the .shapeshifter extract the zip to get it
[lock_to_unlock_anim.zip](https://github.com/alexjlockwood/ShapeShifter/files/1473813/lock_to_unlock_anim.zip)

**Comment by alexjlockwood** [comment]
Oh, I see. Does this have any visual impact on the result? Because the default value for fillAlpha is 1, so even though it isn't explicitly listed, I think the value should still be 1? Or am I missing something?

**Comment by SanthoshDhandapani** [comment]
Yes alex it impacts when i use it android. If i code there with android:fillAlpha="1" everything works as expected.

**Comment by alexjlockwood** [comment]
How does it affect the appearance visually? How are you using the AVD in your Android app?

**Comment by SanthoshDhandapani** [comment]
I tried applying the exported avd with this sample project https://github.com/ANPez/Animated-Vector-Drawables
one more clue is the animation which i used here has fillAlpha animation.
